# -----------------------------------------------------------------------------------
# variable names used in dmp_trajectory.cpp
# -----------------------------------------------------------------------------------

trajectory:

  pool_and_joint_right_arm: r_cue_y r_cue_yd r_cue_ydd r_cue_roll r_cue_rolld r_cue_rolldd r_cue_pitch r_cue_pitchd r_cue_pitchdd r_cue_yaw r_cue_yawd r_cue_yawdd r_shoulder_pan r_shoulder_pan_d r_shoulder_pan_dd r_shoulder_lift r_shoulder_lift_d r_shoulder_lift_dd r_upper_arm_roll r_upper_arm_roll_d r_upper_arm_roll_dd r_elbow_flex r_elbow_flex_d r_elbow_flex_dd r_forearm_roll r_forearm_roll_d r_forearm_roll_dd r_wrist_flex r_wrist_flex_d r_wrist_flex_dd r_wrist_roll r_wrist_roll_d r_wrist_roll_dd

  debug_pool_right_arm: time dt unspecified r_cue_y r_cue_yd r_cue_ydd r_cue_roll r_cue_rolld r_cue_rolldd r_cue_pitch r_cue_pitchd r_cue_pitchdd r_cue_yaw r_cue_yawd r_cue_yawdd
  pool_right_arm: r_cue_y r_cue_yd r_cue_ydd r_cue_roll r_cue_rolld r_cue_rolldd r_cue_pitch r_cue_pitchd r_cue_pitchdd r_cue_yaw r_cue_yawd r_cue_yawdd
  r_cue_y:
    name: r_cue_y
    unit: m
  r_cue_yd:
    name: r_cue_yd
    unit: m/s
  r_cue_ydd:
    name: r_cue_ydd
    unit: m/s^2
  r_cue_roll:
    name: r_cue_roll
    unit: rad
  r_cue_rolld:
    name: r_cue_rolld
    unit: rad/s
  r_cue_rolldd:
    name: r_cue_rolldd
    unit: rad/s^2
  r_cue_pitch:
    name: r_cue_pitch
    unit: rad
  r_cue_pitchd:
    name: r_cue_pitchd
    unit: rad/s
  r_cue_pitchdd:
    name: r_cue_pitchdd
    unit: rad/s^2
  r_cue_yaw:
    name: r_cue_yaw
    unit: rad
  r_cue_yawd:
    name: r_cue_yawd
    unit: rad/s
  r_cue_yawdd:
    name: r_cue_yawdd
    unit: rad/s^2
